numpy/random/__init__.py

```diff
@@ -177,7 +177,6 @@
             'ThreeFry', 'Xoshiro256', 'Xoshiro512', 'RandomState']
 
 # Some aliases:
-ranf = random = sample = random_sample
 __all__.extend(['ranf', 'random', 'sample'])
```

Comment:
Nit: this could get a cleanup and be moved to the huge `__all__` list above, which should get a comment that these are all imported with `from mtrand import *`

Reply:
Fixed.